integrationExamples/gpt/liveIntentId_example.html

```diff
@@ -0,0 +1,69 @@
+<!--
```

Comment:
please drop this file. We need to keep the integration examples clean. This is too difficult for the core team to maintain. Documentation can go into prebid.org if you submit a PR there.